tests/behat/features/create-taxonomies.feature

```diff
@@ -3,7 +3,7 @@ Feature: Create taxonomies
   I want to create and link taxonomies
 
   Background:
-    Given the "group" "EDITOR group" has permissions "CMS_ACCESS_LeftAndMain" and "TAXONOMYTERM_CREATE" and "TAXONOMYTERM_EDIT" and "TAXONOMYTERM_DELETE"
+    Given the "group" "EDITOR" has permissions "CMS_ACCESS_LeftAndMain" and "TAXONOMYTERM_CREATE" and "TAXONOMYTERM_EDIT" and "TAXONOMYTERM_DELETE"
```

Comment:
Surely full `CMS_ACCESS_LeftAndMain` access isn't necessary for every scenario here?

Reply:
DONE